src/main/java/bio/overture/ego/service/VisaService.java

```diff
@@ -65,6 +66,18 @@ public void delete(@NonNull UUID id) {
     super.delete(id);
   }
 
+  // Parses Visa JWT token to convert into Visa Object
+  public Visa parseVisa(@NonNull String visaJwtToken) throws JsonProcessingException {
+    String[] split_string = visaJwtToken.split("//.");
```

Comment:
regex syntax uses backlash to scape special characters. in this case should be like "\\."